charts/incubator/tasmoadmin/Chart.yaml

```diff
@@ -3,8 +3,7 @@ appVersion: "2.4.2"
 dependencies:
   - name: common
     repository: https://library-charts.truecharts.org
-    version: 11.1.2
-deprecated: false
```

Comment:
Why is this removed?

Reply:
No real reason, but I have used code-server (also not set) as ref and since it's optional I thought it can be removed. Sorry though!